mlflow/store/artifact/artifact_repo.py

```diff
@@ -150,6 +150,15 @@ def _download_file(self, remote_file_path, local_path):
         """
         pass
 
+    def delete_artifacts(self, artifact_path):
```

Comment:
Nit: Can we add the `experimental` decorator to this method as well?

Reply:
Done